### PR TITLE
Changes for weston 9.0.91

### PR DESCRIPTION
**Pull request by no92**

### CODE REVIEW

File: options/linux/include/sys/epoll.h [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 // TODO: We only need sigset_t and not the whole signal.h
 #include <stdint.h>
-#include <signal.h>
+#include <abi-bits/signal.h>
```
Comment (Geertiebear) [inline options/linux/include/sys/epoll.h:6]: Why is this change needed? If this fixes the above TODO, could you remove it please?

Comment (no92) [reply options/linux/include/sys/epoll.h:6]: This both fixes the TODO and resolves problems with weston, where includes of this do seem correct.

**Comment by no92** [comment]
Addressed the issue raised.

**Review by Geertiebear — APPROVED** [review]
Thanks, LGTM now!
